introduction.md

```diff
@@ -0,0 +1 @@
+yo...I'm Zach, and I'ma clean up dis town!!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.